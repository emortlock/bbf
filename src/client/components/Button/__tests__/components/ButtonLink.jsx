import React from 'react'
import { shallow } from 'enzyme'

import { ButtonLink } from '../..'

const setup = (testProps = {}) => {
  const props = {
    to: '/',
    ...testProps,
  }

  const wrapper = shallow(<ButtonLink {...props}>Content</ButtonLink>)

  return {
    props,
    wrapper,
  }
}

describe('ButtonLink', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
