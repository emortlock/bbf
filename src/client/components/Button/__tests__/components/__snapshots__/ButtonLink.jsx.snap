// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLink renders matching snapshot 1`] = `
<Link
  href="/"
>
  <a
    className="inline-block bg-white hover:bg-grey-light text-black hover:text-white rounded hover:no-underline py-2 px-4"
  >
    Content
  </a>
</Link>
`;
